angular-client/src/components/history/carousel.component.html

```diff
@@ -0,0 +1,21 @@
+<p-carousel [value]="runs" [numVisible]="1" [numScroll]="1" [circular]="true" [showIndicators]="false">
+  <ng-template let-run pTemplate="item">
+    <div class="runcard">
+      <div class="row">
+        <h3 class="left">Run #{{ run.id }}</h3>
```

Comment:
I think we should make a custom styled component for this and then use it where you use all the left classes